src/services/author.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Repository } from 'typeorm';
+import { AppDataSource } from '../configs/ormConfig';
+import { AuthorEntity } from '../entities/author.entity';
+
+export class AuthorService {
+  private readonly authorRepository: Repository<AuthorEntity>;
+  constructor() {
+    this.authorRepository = AppDataSource.getRepository(AuthorEntity);
+  }
+
+  async getAllAuthors(): Promise<AuthorEntity[]> {
+    const authors = await this.authorRepository.find({
+      order: { firstName: 'ASC' },
+    });
+    return authors;
```

Comment:
chỗ này return luôn, ko cần gán biến đâu e nè, fix tương tự bên dưới nhé
```suggestion
    return await this.authorRepository.find({
      order: { firstName: 'ASC' },
    });
```